src/Cli/Microsoft.DotNet.Cli.Utils/NativeMethods.cs

```diff
@@ -68,29 +72,91 @@ internal struct PROCESS_BASIC_INFORMATION
             public UIntPtr UniqueProcessId;
             public UIntPtr InheritedFromUniqueProcessId;
         }
-
+#if NET
+        [LibraryImport("kernel32.dll", StringMarshalling = StringMarshalling.Utf16, SetLastError = true)]
+#else
         [DllImport("kernel32.dll", CharSet = CharSet.Unicode, SetLastError = true)]
```

Comment:
Can probably add `[DefaultDllImportSearchPaths(DllImportSearchPath.System32)]` to anything from OS.

Reply:
It works, but `kernel32.dll` should be in the [Known DLLs](https://learn.microsoft.com/en-us/windows/win32/dlls/dynamic-link-library-search-order#factors-that-affect-searching) list, and specifying a default search path is not necessary.